src/Microsoft.Diagnostics.Monitoring.WebApi/IExperimentalFlags.cs

```diff
@@ -9,8 +9,5 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi
 {
     internal interface IExperimentalFlags
     {
```

Comment:
Are we intentionally keeping this around as an empty interface?

Reply:
I'm leaving this and the experimental feature logger in for now since parameter capturing are using them and wanted to make that merge a little easier. They can be removed once that feature moves away from this interface.